sycl-fusion/jit-compiler/include/JITContext.h

```diff
@@ -61,7 +65,7 @@ class JITContext {
 
   llvm::LLVMContext *getLLVMContext();
 
-  SPIRVBinary &emplaceSPIRVBinary(std::string Binary);
+  KernelBinary &emplaceSPIRVBinary(std::string Binary, BinaryFormat Format);
```

Comment:
```suggestion
  template <typename... Ts>
  KernelBinary &emplaceSPIRVBinary(Ts&&... Args) {
    return Binaries.emplace_back(std::forward<Ts>(Args)...);
  }
```
Something similar to avoid future updates.